app/config/index.js

```diff
@@ -1,15 +1,82 @@
+const Joi = require('joi')
+
 if (process.env.NODE_ENV !== 'test') {
   require('dotenv').config()
 }
 
+const schema = Joi.object({
+  env: Joi.string().valid('development', 'test', 'production').required(),
+
+  fundingFarmingApiUri: Joi.string().uri().required(),
+  fundingFarmingApiKey: Joi.string().required(),
+  appInsightsKey: Joi.string().optional(),
```

Comment:
This looks good. please edit the docker-compose.yaml file and remove the line where APPINSIGHTS_CONNECTIONSTRING is set then this check shouldn't give false positives

Reply:
done